### PR TITLE
Adding cron for daily tests and builds on Mercodee

### PR DESCRIPTION
**Pull request by dimpar**
The purpose is to set a cron job that would run tests and builds on daily basis so it can be displayed on Meercode dashboard.

### CODE REVIEW

File: .github/workflows/client.yml [hunk 1]
```diff
@@ -44,7 +46,7 @@ jobs:
   client-build-test-publish-ethereum:
     needs: client-detect-changes
     if: |
-      github.event_name == 'push'
+      github.event_name != 'pull_request'
```
Comment (michalinacienciala) [inline .github/workflows/client.yml:49]: We don't want to execute the job for example for `workflow_dispatch` with `alfajores` input. With the proposed config, the job would execute for such workflow.

My suggestion:
```suggestion
      github.event_name == 'push'
        || github.event_name == 'schedule'
```

File: .github/workflows/client.yml [hunk 2]
```diff
@@ -147,7 +149,7 @@ jobs:
   client-build-test-publish-celo:
     needs: client-detect-changes
     if: |
-      github.event_name == 'push'
+      github.event_name != 'pull_request'
```
Comment (michalinacienciala) [inline .github/workflows/client.yml:152]: My suggestion:
```suggestion
      github.event_name == 'push'
        || github.event_name == 'schedule'
```

**Review by michalinacienciala — CHANGES_REQUESTED** [review]

File: .github/workflows/client.yml (hunk 1 above)
Comment (dimpar) [reply .github/workflows/client.yml]: agree, good catch c534da4a963b8b428d8936a9ff7e9ca0c54f1983

File: .github/workflows/client.yml (hunk 2 above)
Comment (dimpar) [reply .github/workflows/client.yml]: c534da4a963b8b428d8936a9ff7e9ca0c54f1983

**Review by michalinacienciala — APPROVED** [review]

**Pull request merged**